.env.test

```
@@ -6,8 +6,8 @@ EMAIL_TRANSPORT="smtp://localhost:1025"
 EMAIL_FROM="test@revisium.local"
 FILE_PLUGIN_PUBLIC_ENDPOINT=http://test-files.localhost
 
-TRANSACTION_MAX_WAIT=5000
-TRANSACTION_TIMEOUT=10000
-TRANSACTION_MAX_RETRIES=50
-TRANSACTION_BASE_DELAY_MS=10
-TRANSACTION_MAX_DELAY_MS=500
+TRANSACTION_MAX_WAIT=10000
+TRANSACTION_TIMEOUT=15000
+TRANSACTION_MAX_RETRIES=30
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TRANSACTION_MAX_RETRIES key should go before the TRANSACTION_MAX_WAIT key

(UnorderedKey)